html/changelogs/changelog2017.html

```diff
@@ -2368,7 +2368,7 @@ <h2 class='date'>22 January 2017</h2>
 	<h3 class='author'>Rahlzel updated:</h3>
 	<ul class='changes bgimages16'>
 		<li class='rscdel'>Cloning system removed!</li>
-		<li class='tweak'>Defibrilators have been buffed significantly. Players dead within 5 minutes can be revived as long as the defibrilator has enough charges, and as long as the player didn't die from suicide, chestbursting, head removal or extreme damage. Defibrilators can be charged using the same rechargers that are used for tasers and stun batons.</li>
+		<li class='tweak'>Defibrilators have been buffed significantly. Players dead within 5 minutes can be revived as long as the defibrillator has enough charges, and as long as the player didn't die from suicide, chestbursting, head removal or extreme damage. Defibrilators can be charged using the same rechargers that are used for tasers and stun batons.</li>
```

Comment:
Do not touch old changelog please. Typos do not matter there

Reply:
Alright sure yeah, thought I reverted those changes already, I'll fix it in a bit